lib/Mail/Milter/Authentication/Constants.pm

```diff
@@ -78,6 +78,7 @@ use constant SMFIP_NOBODY       => 0x10;
 use constant SMFIP_NOHDRS       => 0x20;
 use constant SMFIP_NOEOH        => 0x40;
 use constant SMFIP_NONE         => 0x7F;
+use constant SMFIP_HDR_LEADSPC  => 0x100000;
```

Comment:
Again, format seems to differ from the lines above. I don't have the milter spec to confirm what's correct here, can you point me at something?

Reply:
Sure, I've been using this: https://pymilter.org/pymilter/milter_api/xxfi_negotiate.html

And Postfix's definition of those constants: https://github.com/vdukhovni/postfix/blob/master/postfix/src/milter/milter8.c#L206

I'm happy to format them however you like.